components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/constants/Constants.java

```diff
@@ -51,11 +51,19 @@
     public static final String SEPARATED_WITH_SPACE = " ";
     public static final String ERROR = "error";
     public static final String ERROR_DESCRIPTION = "error_description";
+    public static final String DEVICE_DO_ENDPOINT = "/authenticationendpoint/device.do";
```

Comment:
DEVICE_DO_ENDPOINT Better change to DEVICE_ENDPOINT_PATH
DEVICE_SUCCESS_DO_ENDPOINT Better change to DEVICE_SUCCESS_ENDPOINT_PATH